R/woods_plot.R

```diff
@@ -15,9 +15,10 @@
 #' protein identifiers.
 #' @param colouring Optional argument, column in the data frame containing information by which peptide or precursors should
 #' be coloured.
-#' @param fold_change_cutoff Optional argument specifying the log2 fold change cutoff used for assessing whether changes are significant. The default value is 2.
+#' @param fold_change_cutoff Optional argument specifying the log2 fold change cutoff used in the plot. The default value is 2.
+#' @param highlight Optional column containing logicals, specifying whether specific peptides or precursors should be highlighted with an asterisk. Default is \code{highlight = NULL}.
```

Comment:
```suggestion
#' @param highlight Optional column containing logicals, specifying whether specific peptides or precursors should be highlighted with an asterisk.
```

Would not say that default is NULL because that is implied by the fact that it is optional. If the user does not want to use it they should not provide it.